### DISCUSSION TITLE
Can we add Text Field from Material UI design ?

### DISCUSSION THREAD

**Discussion by ghostlexly** [shadcn-ui/ui#894] [Ideas]
Hey,

Can we add Text Field from Material UI design ?
Including the outlined variants, password, icons inside inputs..

**Comment by nordinvandijk** [comment]
Just use MUI if you want MUI. shadcn/ui is just a uniform styling and better dx applied over great headless libraries. That uniform styling is not Material Design. So it would make no sense.
